README.md

```diff
@@ -16,36 +16,54 @@ This module includes a a set of pre-defined rules for commonly used protocols (f
 The following example demonstrate how to use the network-security-group module with a combination of predefined and custom rules.
 
 ```hcl
-resource "azurerm_resource_group" "test" {
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
   name     = "my-resources"
   location = "West Europe"
 }
 
+resource "azurerm_application_security_group" "first" {
+  name                = "asg-first"
+  location            = "eastus"
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+resource "azurerm_application_security_group" "second" {
+  name                = "asg-second"
+  location            = "eastus"
+  resource_group_name = azurerm_resource_group.example.name
+}
+
 module "network-security-group" {
-  source                = "Azure/network-security-group/azurerm"
-  resource_group_name   = azurerm_resource_group.test.name
-  location              = "EastUS" # Optional; if not provided, will use Resource Group location
-  security_group_name   = "nsg"
-  source_address_prefix = ["10.0.3.0/24"]
```

Comment:
It's great to add example of asg_ids in this readme. While may I suggest we just add this example rather than overriding the previous x_address_prefix example? I.e. I would expect there are two examples here instructing users how to play with their different purpose.